catalog/transformer/catalog-transformer-tika-input/src/test/java/ddf/catalog/transformer/input/tika/TikaInputTransformerTest.java

```diff
@@ -653,18 +660,80 @@ private String convertDate(Date date) {
     }
 
     private Metacard transform(InputStream stream) throws Exception {
-        TikaInputTransformer tikaInputTransformer = new TikaInputTransformer(null,
-                getCommonMetacardType());
         Metacard metacard = tikaInputTransformer.transform(stream);
         return metacard;
     }
 
     private static MetacardType getCommonMetacardType() {
-        return new MetacardTypeImpl(COMMON_METACARDTYPE, Arrays.asList(new ValidationAttributes(),
-                new ContactAttributes(),
-                new LocationAttributes(),
-                new MediaAttributes(),
-                new AssociationsAttributes()));
+        return new MetacardTypeImpl(COMMON_METACARDTYPE_NAME,
```

Comment:
It looks like you could extract the repeated body of these methods into a shared method that just takes the metacard type name constant as input.